Makefile

```diff
@@ -11,7 +11,7 @@ BUILD_LDFLAGS := "-X main.revision=$(GIT_REVISION)"
 
 SOURCES = $(shell find . -type f -name '*.go')
 
-all: test $(OUTPUT_DIR)/$(NAME)
+all: $(OUTPUT_DIR)/$(NAME)
```

Comment:
**Dockerfile** executes `make all` with specifying *NAME* variable:
https://github.com/mackerelio-labs/mackerel-sql-metric-collector/blob/374b88ebf1cb5f987022254e1b42a8d23f663875/Dockerfile#L4

*build* workflow runs after _test_ workflow so it do not need to run `go test` on building the docker image.
https://github.com/mackerelio-labs/mackerel-sql-metric-collector/blob/374b88ebf1cb5f987022254e1b42a8d23f663875/.github/workflows/ci.yaml#L37-L39